test/files/pos/lazyref-autoapply.scala

```diff
@@ -0,0 +1,8 @@
+// scalac: -Xsource:2.14
```

Comment:
This will not fail the test without `-Werror`. `pos` don't have check files.

Reply:
(fixed in https://github.com/scala/scala/pull/8743)